common/PlayAreaScreen.tsx

```diff
@@ -27,15 +29,22 @@ const styles = StyleSheet.create({
   },
 });
 
+// flag for sound in checkers every move include two clicks
+let checkersFlag = true;
+// backround suond flag
+let backGroundMusic = false;
+
 export function PlayAreaScreen() {
+  const [sound, setSound] = React.useState<Sound | undefined>(undefined);
+  const [backroundSound, setBackgroundSound] = React.useState<Sound | undefined>(undefined);
```

Comment:
This means you only have background sound in the play area, and it should be all the time.
You need to add this at Main.

Reply:
ok I will change it

Reply:
I want to have the new UI setting screen to add it there,
can you merge it and I will fix it later?